src/dotnet/Chat/Markup/Mention.cs

```diff
@@ -14,6 +14,7 @@ public override string Format()
 
     public string Format(MentionFormat format)
         => format switch {
+            MentionFormat.NameOnly => $"@{Name.NullIfEmpty() ?? Id}",
```

Comment:
Check out `_` clause - it is actually responsible for this.

Reply:
For the note, NameOnly means we never show Ids, so my code in the end is right. IMO it's fine to quote the string all the times here for now, btw, since the names are going to be with spaces, etc. - in short, let's solve nice quoting later.

Reply:
fixed